lang/gapi.go

```diff
@@ -110,11 +110,11 @@ func (obj *GAPI) CliFlags(command string) []cli.Flag {
 	case gapi.CommandDeploy:
 		flags := []cli.Flag{
 			// TODO: removed (temporarily?)
-			//cli.BoolFlag{
+			//*cli.BoolFlag{
 			//	Name:  "stdin",
 			//	Usage: "use passthrough stdin",
 			//},
-			modulePath,
+			&modulePath,
```

Comment:
actually I think you want the & up higher in the modulePath definition.

eg:  	`modulePath := cli.StringFlag{` becomes:

` 	modulePath := &cli.StringFlag{`

Reply:
Ah. Yes, this is silly. It was one of the first places I touched and had not yet grokked this.